Sources/PointFree/Collections/CollectionSection.swift

```diff
@@ -0,0 +1,49 @@
+import HttpPipeline
+import Models
+import PointFreePrelude
+import Prelude
+import Tuple
+import Views
+
+let collectionSectionMiddleware
+  : M<Tuple4<User?, SubscriberState, Episode.Collection.Slug, Episode.Collection.Section.Slug>>
+  = basicAuth(
+    user: Current.envVars.basicAuth.username,
+    password: Current.envVars.basicAuth.password
+    )
+    <<< fetchCollectionSectionMiddleware
+    <| map(lower)
+    >>> writeStatus(.ok)
+    >=> respond(
+      view: collectionSection,
+      layoutData: { currentUser, _, collection, section in
+        SimplePageLayoutData(
+          currentUser: currentUser,
+          data: (collection, section),
+          extraStyles: collectionIndexStylesheet,
```

Comment:
we should add a trello card or a todo somewhere to track that we need to add meta tags to all of our new pages